src/main/resources/assets/ironbackpacks/lang/zh_CN.lang

```diff
@@ -1,5 +1,43 @@
 #PARSE_ESCAPES
 
+# Creative Tab
+itemGroup.ironbackpacks=特殊背包
+
+# Items
+item.ironbackpacks.backpack.name=破损背包
+item.ironbackpacks.backpack.ironbackpacks.basic.name=基础背包
+item.ironbackpacks.backpack.ironbackpacks.iron.name=铁背包
+item.ironbackpacks.backpack.ironbackpacks.gold.name=金背包
+item.ironbackpacks.backpack.ironbackpacks.diamond.name=钻石背包
+item.ironbackpacks.upgrade.name=空白升级
+
+# Tooltips
+tooltip.ironbackpacks.upgrade.cost=需要消耗%d升级点数（s）
+tooltip.ironbackpacks.upgrade.minimum_tier=最低等级：%d
+tooltip.ironbackpacks.backpack.upgrade.list=已安装的升级：
+tooltip.ironbackpacks.shift=按住Shift显示更多信息
+
+# Upgrades
+upgrade.ironbackpacks.damage_bar.name=精准升级
+upgrade.ironbackpacks.damage_bar.desc=添加一个统计数据，精准的反应背包内的一切。
+upgrade.ironbackpacks.lock.name=私有升级
+upgrade.ironbackpacks.lock.desc=锁定背包，但不包括所有者。
+upgrade.ironbackpacks.extra_upgrade.name=附加升级
+upgrade.ironbackpacks.extra_upgrade.desc=添加一个额外的升级点数。
+
+# GUI
+container.ironbackpacks.backpack=打开背包
+
+# JEI
+# Increase Tier
+jei.ironbackpacks.increaseTier.name=升级背包
+jei.ironbackpacks.increaseTier.desc=要升级背包，只需要按照§2下面的方法合成§r。 在两种类型的背包中做出选择，请三思而后行！
```

Comment:
Extra space after period, remove it

Reply:
Ignore this... for weird reason they have to be here to ensure correct line break.